systemtest/src/test/java/io/strimzi/systemtest/ConnectS2IST.java

```diff
@@ -209,9 +211,17 @@ void testSecretsWithKafkaConnectS2IWithTlsAndScramShaAuthentication() {
         KafkaClientsResource.deployKafkaClients(true, CLUSTER_NAME + "-" + Constants.KAFKA_CLIENTS, user).done();
 
         final String defaultKafkaClientsPodName =
-                ResourceManager.kubeClient().listPodsByPrefixInName(CLUSTER_NAME + "-" + Constants.KAFKA_CLIENTS).get(0).getMetadata().getName();
-
-        internalKafkaClient.setPodName(defaultKafkaClientsPodName);
+            ResourceManager.kubeClient().listPodsByPrefixInName(CLUSTER_NAME + "-" + Constants.KAFKA_CLIENTS).get(0).getMetadata().getName();
+
+        InternalKafkaClient internalKafkaClient = new InternalKafkaClient.Builder()
+            .withUsingPodName(defaultKafkaClientsPodName)
+            .withTopicName(CONNECT_S2I_TOPIC_NAME)
+            .withNamespaceName(NAMESPACE)
+            .withClusterName(CLUSTER_NAME)
+            .withMessageCount(MESSAGE_COUNT)
+            .withSecurityProtocol("TLS")
+            .withConsumerGroupName("my-group-" + new Random().nextInt(Integer.MAX_VALUE))
```

Comment:
sure, we can create some issue for that as enhancement ))

Reply:
anyway, it would be better just to encapsulate this `consumer-group-name` and have two options:
1) if specified create that with the concrete name
2) if not specify create the random one

Reply:
Agree